patterns/vsi-extension/version.tf

```diff
@@ -6,9 +6,10 @@ terraform {
   required_version = ">= 1.3, < 1.6"
   # Pin to the lowest provider version of the range defined in the main module's version.tf to ensure lowest version still works
   required_providers {
+    # Pin to 1.58.1 due to https://github.com/IBM-Cloud/terraform-provider-ibm/issues/4907
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "1.59.0"
+      version = "1.58.1"
```

Comment:
FYI, renovate is going to update this back to the latest due to the rules at https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/blob/workspace/renovate.json

Are we sure that this only happens in 1.59.0 and not in 1.58.1 ?

Reply:
Yes, I have created a provider issue as well for it.

Reply:
https://github.com/IBM-Cloud/terraform-provider-ibm/issues/4907

Reply:
OK, so we need to update the renovate rules to prevent it from updating this. Its currently set up to update all of the directories in `"matchPaths": ["patterns"],` so we will need to exclude the `patterns/vsi-extension` dir